Userland/Applications/Browser/BrowserWindow.cpp

```diff
@@ -162,7 +162,8 @@ void BrowserWindow::build_menus(StringView const man_file)
     file_menu->add_separator();
     file_menu->add_action(GUI::CommonActions::make_quit_action([](auto&) {
         GUI::Application::the()->quit();
-    }));
+    },
+        false));
```

Comment:
The formatting was done that way in order to appease the linter.

Reply:
You can do
```cpp
file_menu->add_action(GUI::CommonActions::make_quit_action(
    [](auto&) {
        GUI::Application::the()->quit();
    },
    false));
```

Reply:
I have tried what you suggested, and Meta/lint-clang-format.sh insists on the ugly indentation, otherwise it fails.

Reply:
That's strange because I tested the formatting inline inside of `BrowserWindow.cpp` and everything was fine. Are you sure your clang-format version is up-to-date (18)?

Reply:
Looks like it was just a copy/paste issue on my part. Your suggestion is working correctly now, and I have applied it to any files that previously had the ugly indentation.